src/openforms/prefill/contrib/family_members/tests/test_filters.py

```diff
@@ -0,0 +1,308 @@
+from django.test import TestCase
+
+from freezegun import freeze_time
+
+from openforms.contrib.haal_centraal.constants import (
+    NaturalPersonDetails as HC_NaturalPersonDetails,
+)
+from stuf.stuf_bg.constants import NaturalPersonDetails as StUFBG_NaturalPersonDetails
+
+from ..filters import filter_members_by_age
+
+
+@freeze_time("2025-04-25T18:00:00+01:00")
+class FamilyMembersFiltersTests(TestCase):
+    def setUp(self) -> None:
+        super().setUp()
+
+        self.HC_DATA = [
+            HC_NaturalPersonDetails(
+                bsn="123456788",
+                first_names="Doe",
+                lastname="Pirsen",
+                date_of_birth="2007-01-19",
+                date_of_birth_precision="date",
+            )
+        ]
+
+        self.STUF_BG_DATA = [
+            StUFBG_NaturalPersonDetails(
+                bsn="123456788",
+                first_names="Doe",
+                lastname="Pirsen",
+                date_of_birth="2007-01-19",
+            ),
+        ]
```

Comment:
setting up test data in `setUp` that's modified inside each tests or not used/relevant for a test at all is a complete anti-pattern. Set up the necessary test data at the beginning of the test, and then assert on it instead.

For the irrelevant fields, you can use factory boy - it's not just for Django models.